docs/build_docs_locally.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/sh
-
```

Comment:
add a note about this to the docs README, how to invoke it

Reply:
Moved to README